### PR TITLE
Read ELASTICSEARCH_INDEX_PREFIX configuration variable

### PR DESCRIPTION
**Pull request by seitztimo**
Configuration variable to allow changing Elasticsearch index names. Environmental variable ELASTICSEARCH_INDEX_PREFIX is used to construct index names for different languages. This does not need any configuration changes for services that does not need to use different index names.

### CODE REVIEW

File: smbackend/settings.py [hunk 1]
```diff
@@ -14,6 +14,7 @@
     LANGUAGES=(list, ["fi", "sv", "en"]),
     DATABASE_URL=(str, "postgis:///servicemap"),
     ELASTICSEARCH_URL=(str, None),
+    ELASTICSEARCH_INDEX_PREFIX=(str, 'servicemap'),
```
Comment (mhieta) [inline smbackend/settings.py:17]: Current standard black-formatting rules prefer double-quotes instead of single.

**Review by mhieta — CHANGES_REQUESTED** [review]
Added one minor comment. Looks good otherwise 👍

File: smbackend/settings.py (hunk 1 above)
Comment (seitztimo) [reply smbackend/settings.py]: Oh yes. I should have noticed that. IT is now changed in commit 012e53ab53a4902e009bdf43a1e015f9a140bf67

**Review by mhieta — APPROVED** [review]
Looks good now 👍

**Pull request merged**